api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPTEligiblePatientListCohortQueries.java

```diff
@@ -3419,19 +2974,17 @@ public CohortDefinition get3HPLastProfilaxyDuringM3Period(sectionFromM section)
             + "                   WHERE  p.voided = 0 "
             + "                          AND e.voided = 0 "
             + "                          AND o.voided = 0 "
-            + "                          AND o2.voided = 0 "
             + "                          AND e.encounter_type = ${53} "
             + "                          AND (o.concept_id = ${23985} "
             + "                          AND o.value_coded = ${23954}) "
             + "                          AND e.location_id = :location "
             + "                          AND e.encounter_datetime <= :endDate "
-            + "                          AND ( o2.concept_id = ${6129} "
-            + "                          AND o2.value_datetime IS NOT NULL "
```

Comment:
should the end data verification not be verified using a new structure? Value_coded = Fim (id= 1267) for concept_id = 165308. Check the requirements